lua/ttt2/libraries/marker_vision.lua

```diff
@@ -227,6 +228,24 @@ if CLIENT then
         { font = "Tahoma", size = 14, weight = 300, extended = true }
     )
 
+    ---
+    -- This gets the currently focussed closest entity.
+    -- @realm client
+    function markerVision.GetFocussedEntity()
+        local closestEntTbl = markerVision.focussedMarkers[1] or {}
+
+        for i = 2, #markerVision.focussedMarkers do
+            local entTbl = markerVision.focussedMarkers[i]
+            if entTbl.screenDistanceSquared < closestEntTbl.screenDistanceSquared then
+                closestEntTbl = entTbl
+            end
+        end
+
+        markerVision.focussedMarkers = { closestEntTbl }
+
+        return closestEntTbl.entity
+    end
```

Comment:
just a note; this could probably unnecessary if we implemented a marker sorting system on a frame by frame base that makes sure they are also rendered in the correct order (which they aren't right now)

Reply:
Exactly, or what I recently did with the overhead icons

Reply:
I guess you won't implement a sorting for the rendering in this PR?

Reply:
Not really, I only get the foremost marker, therefore the interface/API stands and if we wanna sort it at one point internally and just return the foremost marker inside this function without the sorting its fine as well.